rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -93,21 +94,24 @@ def _toJSON(mol):
 
 
 def _wrapHTMLIntoTable(html):
-  return InteractiveRenderer.injectHTMLHeaderBeforeTable(
+  return InteractiveRenderer.injectHTMLFooterAfterTable(
     f'<div><table><tbody><tr><td style="width: {molSize[0]}px; ' +
     f'height: {molSize[1]}px; text-align: center;">' + html.replace(" scoped", "") +
     '</td></tr></tbody></table></div>')
 
 
 def _toHTML(mol):
+  def escape_special_chars(s):
+    return html.escape(str(s)).replace("&", "&amp;")
```

Comment:
I don't think this is correct. 

Escaping the &s in the results from `html.escape` gives the following in the notebook:
![image](https://user-images.githubusercontent.com/540511/194735487-410fb586-e7be-424c-be9e-63f41fdd228a.png)

On the other hand, if I just use:
```
return html.escape(str(s)) 
```
I get:
![image](https://user-images.githubusercontent.com/540511/194735456-01838e62-7a4c-4b7d-a946-bfafab7c623a.png)

Reply:
@greglandrum When I saw the final `.replace("&", "&amp;")` in the `escape_special_chars()` function I had the feeling it was wrong, but then the 'If I wrote it that way, that must have been a reason' feeling prevailed, while it looks like it was just wrong. I wrote that piece of code ~2 months ago and honestly I can't remember why I introduced that further level of escaping at the time. Thanks for noticing and testing that!